feature_engine/discretisation/arbitrary.py

```diff
@@ -106,6 +103,7 @@ def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
 
         return self
 
+
     def transform(self, X: pd.DataFrame) -> pd.DataFrame:
```

Comment:
we need to delete the transform method entirely. It is directly inherited from the parent class.